.gitlab-ci.yml

```diff
@@ -37,6 +37,33 @@ trigger_child_pipeline:
     strategy: depend
   needs:
     - generate_matrix
+  variables:
+      PARENT_PIPELINE_ID: $CI_PIPELINE_ID
+
+check-big-regressions:
+  stage: benchmarks
+  needs:
+    - job: trigger_child_pipeline
+      artifacts: true
+  when: on_success
+  tags: 
+    - "arch:amd64"
+  image: $BASE_CI_IMAGE
+  script:
+    - |
+      export ARTIFACTS_DIR="$(pwd)/reports" && (mkdir "${ARTIFACTS_DIR}" || :)
+      git config --global url."https://gitlab-ci-token:${CI_JOB_TOKEN}@gitlab.ddbuild.io/DataDog/".insteadOf "https://github.com/DataDog/"
+      git clone --branch dario.castane/APMSP-2045/perf-quality-gate https://github.com/DataDog/benchmarking-platform platform && cd platform
```

Comment:
is there a reason we have to have this script here, rather than adding on to the existing script in `.gitlab/benchmarks.yml`? I doubt it will make a significant difference, but we can save time by cloning once instead of twice.

Reply:
@hannahkm `.gitlab/benchmarks.yml` is running specific benchmarks, and we need to collect the results from each parallelized job to do the analysis, as it needs to be a single gate job.